### ISSUE TITLE
Develop Machine Learning Models for Object and Intruder Classification

### ISSUE THREAD

**Issue by hamzariffic** [hamzariffic/Smart-HomeGuard#3]
Create machine learning models to classify individuals and objects around the building perimeter, focusing on intruder detection.

- [x] Gather and preprocess relevant dataset for model training.
- [x] Develop and train machine learning models using TensorFlow/Keras.
- [ ] Evaluate and fine-tune models for accuracy and reliability.

**Comment by hamzariffic** [comment]
Won't be necessary yet. We have AutoML, RapidAPI and Huggingface

**Comment by hamzariffic** [comment]
Now OpenAI and Gemini are around. We'll see what to do here
